src/index.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable import/no-cycle */
+import './style.css';
+import { Task } from './Task';
+import { TaskList } from './tasklist';
+
+const NewTaskList = new TaskList(JSON.parse(localStorage.getItem('library')) || []);
```

Comment:
Kindly extract TaskList parameters into a single variable as this will make your code cleaner